en/docs/learn/consume-api/account-management-dev-portal/recover-dev-portal-password.md

```diff
@@ -0,0 +1,18 @@
+# Recover Password for the Developer Portal
```

Comment:
I don't think this needs to be documented either.

Reply:
Decided to keep it based on an offline discussion with Nuwan